app/analysis-platform/data/AnalysisPlatformDS/instances/AnalysisPlatform.json

```diff
@@ -2,5 +2,6 @@
   "_id": "4483c9b0-d505-46c9-a157-94c79f4d7b7b",
   "type": "AnalysisPlatformDS/models/AnalysisPlatform",
   "label": "Analysis Platform App",
-  "urlPath": ""
+  "urlPath": "ap",
```

Comment:
```suggestion
```
I believe the `urlPath` property is obsolete now. @soofstad ?

Reply:
good point!
will remove urlpath and the /ap url prefix from the app